docs/access/vscode.md

```diff
@@ -11,6 +11,8 @@ There are two ways to set up the connection:
 The main challenge with using VSCode is that the most convenient method for starting a remote session is to start a remote tunnel from the VS Code GUI.
 This approach starts a session in the standard login environment on that node, however this won't work if you want to be developing in a container, in a uenv, or on a compute node.
 
+This process is also demonstrated in a webinar on [Interactive computing on "Alps"](https://www.cscs.ch/publications/tutorials/2025/video-of-the-webinar-interactive-computing-on-alps). Video and slides accessible.
```

Comment:
How about directly embedding the video, at the time stamp where VS Code is discussed?

Here is an example of embedding a video:
https://docs.cscs.ch/storage/longterm/#further-documentation

I think it should be enough to embed this:
```
<iframe width="100%"
        height="315"
        src="https://www.youtube.com/embed/cLVpJO_fE6I?si=bTmmsS_9QvTHpUqK&amp;start=2257"
        title="YouTube video player"
        frameborder="0"
        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
        referrerpolicy="strict-origin-when-cross-origin"
        allowfullscreen>
</iframe>
```